mobile-app/lib/ui/views/learn/widgets/description/description_widget_view.dart

```diff
@@ -30,33 +30,42 @@ class DescriptionView extends StatelessWidget {
                 crossAxisAlignment: CrossAxisAlignment.start,
                 children: [
                   if (instructions.isNotEmpty)
-                    Padding(
-                      padding: const EdgeInsets.all(8.0),
-                      child: Column(
-                        crossAxisAlignment: CrossAxisAlignment.start,
-                        children: [
-                          Container(
-                            padding: const EdgeInsets.fromLTRB(16, 16, 16, 8),
-                            child: const Text(
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
```

Comment:
Why the extra `Column` here as the same `Column` before the `if` does the same thing

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient